test-suite/djinni/relative_paths.djinni

```diff
@@ -1,3 +1,3 @@
-@import "./date.djinni"
-@import "../djinni/date.djinni"
-@extern "./date.yaml"
+@import "./vendor/third-party/date.djinni"
```

Comment:
This will need to be tweaked to merge conflicts.  Note that some changes were made here to do relative-path testing using a different input file (not date/duration), since the Python code doesn't support external types yet, but wants to be able to test using all.djinni.